components/Badge/Badge.css

```diff
@@ -0,0 +1,19 @@
+.Badge {
+  flex-direction: row;
```

Comment:
Whoops! You forget to define `display: flex;` here, so in reality none of these properties are affecting the icon.

display block and text-align center for the label work fine here, you can just delet lines 2-4 and have no issue: 👍